formstack-baton-requests/src/main/scala/com/gu/identity/formstackbatonrequests/services/FormstackService.scala

```diff
@@ -79,24 +79,33 @@ object FormstackService extends FormstackRequestService with LazyLogging {
     submissionIdEmails: List[SubmissionIdEmail],
     accountToken: FormstackAccountToken,
     encryptionPassword: String): Either[Throwable, List[Submission]] = {
-    submissionIdEmails.traverse { submissionIdEmail =>
+    val submissionResults: Either[Throwable, List[Option[Submission]]] = submissionIdEmails.traverse { submissionIdEmail =>
       val response =
         Http(s"https://www.formstack.com/api/v2/submission/${submissionIdEmail.submissionId}.json")
           .header("Authorization", accountToken.secret)
           .param("encryption_password", encryptionPassword)
           .asString
 
-      if(!response.is2xx) {
-        logger.error(response.body)
-      }
+      def isErrorResponse = !response.is2xx
+
+      // Define errors that we can safely skip
+      def isDecryptionError = response.body.contains("An error occurred while decrypting the submissions")
+      def isSubmissionDeletedError = response.body.contains("An error occurred while decrypting the submissions")
```

Comment:
this is probably the wrong error message for this, right ?

Reply:
Docs seem to [indicate that it'll 404 in this case](https://developers.formstack.com/reference/submission-id-get), and the error will look like:

```json
{
  "status": "error",
  "error": "A valid submission id was not supplied"
}
```